plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakePageSinkProvider.java

```diff
@@ -151,6 +157,12 @@ public ConnectorMergeSink createMergeSink(ConnectorTransactionHandle transaction
         DeltaLakeMergeTableHandle merge = (DeltaLakeMergeTableHandle) mergeHandle;
         DeltaLakeInsertTableHandle tableHandle = merge.getInsertTableHandle();
         ConnectorPageSink pageSink = createPageSink(transactionHandle, session, tableHandle, pageSinkId);
+        boolean cdfEnabled = changeDataFeedEnabled(tableHandle.getMetadataEntry());
+        ColumnMappingMode columnMappingMode = getColumnMappingMode(tableHandle.getMetadataEntry());
+        if (cdfEnabled && columnMappingMode != NONE) {
+            // TODO Support CDF for tables with 'id' and 'name' column mapping
```

Comment:
do we have an issue for this?

Reply:
I don't think #16600 supports CDF. Filed #16967. I will add the link to TODO comment.